Sparkle/SUPKGUnarchiver.m

```diff
@@ -0,0 +1,55 @@
+//
+//  SUPKGUnarchiver.m
+//  Sparkle
+//
+//  Created by Thomas Schmitt
+//  Copyright 2013-2019 Thomas Schmitt. All rights reserved.
+//
+
+#import "SUPKGUnarchiver.h"
+#import "SUUnarchiverNotifier.h"
+#import "SULog.h"
+
+
+@interface SUPKGUnarchiver ()
+
+@property (nonatomic, copy, readonly) NSString *archivePath;
+
+@end
+
+@implementation SUPKGUnarchiver
+@synthesize archivePath = _archivePath;
+
+
++ (BOOL)canUnarchivePath:(NSString *)path
+{
+	return [path.pathExtension isEqualToString:@"pkg"];
+}
+
++ (BOOL)mustValidateBeforeExtraction
+{
+    return NO;
```

Comment:
Isn't that a security vulnerability that bypasses Sparkle's signature checks?

Reply:
Don't think so, because PKGs are checked later. Sparkle gave an error message when I return YES. It also gave an error when signature check fails (hmm .... this was in before the new EdDSA key ... I check this again)

Reply:
Can you add a unit test that proves its correct? (a test that passes with bare pkg signed, and a test that fails with bare pkg when signature is invalid)